server/test/auth.spec.js

```diff
@@ -0,0 +1,224 @@
+require("dotenv").config({ path: __dirname + "/../sample.env" });
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const sinon = require("sinon");
+const { app } = require("../app.js");
+const User = require("../models/User");
+
+chai.should();
+chai.use(chaiHttp);
+
+let agent;
+let clock;
+
+const existingUser = {
+  username: "test",
+  email: "test@gmail.com",
+  invalidEmail: "testgmail.com",
+  password: "123123",
+  wrongPassword: "wrongPassword"
+};
+
+const nonExistingUser = {
```

Comment:
You can call this `newUser`. Just a better naming style, though!

Reply:
Applied